src/components/Header.astro

```diff
@@ -18,11 +18,12 @@ const { main }: { main: NavigationLink[] } = menu;
 <header class="bg-zinc-50 dark:bg-zinc-900">
   <div class="mx-auto max-w-screen-lg px-4 py-8 sm:px-6 sm:py-12 lg:px-8">
     <div class="flex items-start justify-between gap-2 sm:gap-4 md:items-center">
-      <a href="/" class="logo flex items-center" aria-current="page">
+      <a href="/" class="logo flex items-center" aria-current="page" title={config.site.title}>
```

Comment:
Is it necessary to include a 'title' attribute in all <a> elements, Can you provide any documentation regarding the importance of using the 'title' attribute?